app/graphql/types/album_type.rb

```diff
@@ -0,0 +1,8 @@
+module Types
+  class AlbumType < BaseObject
+    field :userId, ID, null: false
```

Comment:
You should only use snake_case for field names, the gem will translate directly to camelCase.